app/cdap/components/WranglerV2/DirectivePanel/DirectivePanel.stories.tsx

```diff
@@ -0,0 +1,85 @@
+import React from 'react';
+import { action } from '@storybook/addon-actions';
+import styled from 'styled-components';
+import { Box } from '@material-ui/core';
+import DirectivePanel from 'components/WranglerV2/DirectivePanel';
+
+export default {
+  title: 'DirectivePanel',
+  component: DirectivePanel,
+};
+
+const handleSearchResultClick = () => action('clicked')('handleSearchResultClick');
+
+// below styled component is just for testing purpose
+const DirectiveWrapper = styled(Box)`
+  margin-top: 400px;
+`;
+
+export function Default(args) {
+  return (
+    <DirectiveWrapper>
+      <DirectivePanel {...args} />
+    </DirectiveWrapper>
+  );
+}
+
+Default.args = {
+  directiveSuggestion: {
+    usage: 'parse-as-log :column format',
+    link: 'https://cdap.atlassian.net/wiki/spaces/DOCS/overview',
+  },
+  resultsList: [
```

Comment:
plz move this resultList value to a separate variable and assign it here
I think it might be useful if we want to write any tests going forward